src/pages/juniorPromise/JuniorPromisePage.tsx

```diff
@@ -1,24 +1,96 @@
+import { TempLogoIc, AlarmIc } from '@assets/svgs';
+import { Header } from '@components/commons/Header';
+import Nav from '@components/commons/Nav';
 import SeniorCard from '@components/commons/seniorCard/SeniorCard';
+import { SENIOR_LIST } from '@components/commons/seniorCard/seniorCardConstants';
 import styled from '@emotion/styled';
 import { useState } from 'react';
 import { BottomSheet } from './components/BottomSheetBg';
 import { SeniorListBackground } from './components/SeniorListBackground';
-import { SENIOR_LIST } from '../../components/commons/seniorCard/seniorCardConstants';
 
 const JuniorPromisePage = () => {
   const { seniorList } = SENIOR_LIST;
-  const [isSheetOpen, setSheetOpen] = useState(false);
-  const handleSheetOpen = () => {
-    setSheetOpen(true);
+  // 필터 버튼
+  const [filterActiveBtn, setFilterActiveBtn] = useState('계열');
+  // 바텀 시트 여는 동작
+  const [isBottomSheetOpen, setIsBottomSheetOpen] = useState(false);
+
+  // 필터 버튼에 정보 넣기, 바텀시트 열기
+  const handleFilterActiveBtn = (btnText: string) => {
+    setFilterActiveBtn(btnText);
+    setIsBottomSheetOpen(true);
+  };
+  // 바텀시트 닫기
+  const handleCloseBottomSheet = () => {
+    setIsBottomSheetOpen(false);
+  };
+
+  // 바텀시트 내 직무 칩
+  const [selectedPosition, setSelectedPosition] = useState(Array(21).fill(false));
+
+  // 선택직무 리스트
+  const arrPosition = [...selectedPosition];
+
+  const handleChipPosition = (positionId: number) => {
+    arrPosition[positionId] = !arrPosition[positionId];
+
+    setSelectedPosition(arrPosition);
+  };
+  // 바텀시트 내 계열 칩
+  const [selectedField, setSelectedField] = useState(Array(7).fill(false));
+
+  // 선택 계열 리스트 T/F
+  const arrField = [...selectedField];
+
+  const handleChipField = (fieldId: number) => {
+    arrField[fieldId] = !arrField[fieldId];
+
+    setSelectedField(arrField);
+  };
+
+  // 초기화 함수
+  const handleReset = () => {
+    setSelectedPosition(Array(21).fill(false));
+    setSelectedField(Array(7).fill(false));
+  };
+
+  // 선택된 직무 칩 수
+  const getPositionTrueNum = (arrPosition: boolean[]) => {
+    return arrPosition.filter((n) => n === true).length;
```

Comment:
```suggestion
    return arrPosition.filter((n) => n).length;
```

p5) true는 ===으로 검사 안해주고 값만 바로넣어줘도 됩니다!!

Reply:
아하,, 오.... 그렇군요! 반영완료했습니다!! 감사합니다!!